assets/benchmark_network_architectures.py

```diff
@@ -1,135 +1,55 @@
 COUPLING_SETTINGS_TEST = {
-    't_args': {
-        'dense_args': dict(units=8, kernel_initializer='glorot_uniform', activation='elu'),
-        'num_dense': 1,
-        'spec_norm': True
+    "t_args": {
+        "dense_args": dict(units=8, kernel_initializer="glorot_uniform", activation="elu"),
+        "num_dense": 1,
+        "spec_norm": True,
+    },
+    "s_args": {
+        "dense_args": dict(units=16, kernel_initializer="glorot_uniform", activation="elu"),
+        "num_dense": 1,
+        "spec_norm": False,
     },
-    's_args': {
-        'dense_args': dict(units=16, kernel_initializer='glorot_uniform', activation='elu'),
-        'num_dense': 1,
-        'spec_norm': False
-    }
 }
 
 NETWORK_SETTINGS = {
-    'gaussian_linear': {
-        'posterior': {
-            'num_params': 10,
-            'num_coupling_layers': 2,
-            'coupling_net_settings': COUPLING_SETTINGS_TEST
-        },
-        'likelihood': {
-            'num_params': 10,
-            'num_coupling_layers': 2,
-            'coupling_net_settings': COUPLING_SETTINGS_TEST
-        }
+    "gaussian_linear": {
```

Comment:
Note: This contraction into 1 line (which is possible due to 120 char limit, probably not with 79) can be avoided by explicitly setting a comma (",") also after the last entry, in which case the previous formatting with a single entry per line will be used.